__tests__/app.test.js

```diff
@@ -217,4 +219,59 @@ describe("POST /api/articles/:article_id/comments",()=>{
             expect(response.body.msg).toBe('Bad Request');
         });
     })
+})
+describe("PATCH /api/articles/:article_id",()=>{
+    test("updates the article, responds with 200 status and the updated Article",()=>{
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes: -100})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String)
+            })
+        });
+    })
+    test('responds with status 400 and error message Bad Request when body passed has wrong data type', () => {
+        return request(app)
+        .patch('/api/articles/2')
+        .send({ inc_votes: 'votes' })
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad Request');
+        });
+    })
+    test('responds with status 400 and error message Bad Request when fields/body are missing/malformed', () => {
```

Comment:
This is fine for now - but if you want to make a patch to an article for a different reason in the future, you may have to set this as a 200 and just not do anything with it (or have some other kind of logic in the model)